harness/tests/experiment/keras/test_tf_keras_trial.py

```diff
@@ -340,7 +340,7 @@ def test_surface_native_error():
             assert (
                 b"ValueError: Shapes (None, 10) and (None, 1) are incompatible" in err
                 or b"ValueError: Input 0 of layer sequential is incompatible with the "
-                b"layer: : expected min_ndim=2, found ndim=1. Full shape received: [1]" in err
+                b"layer: : expected min_ndim=2, found ndim=1. Full shape received: (1,)" in err
```

Comment:
Actually you can make this work with both the old and new versions. This is already a list of possible strings to find in the error. We should add the slightly different possibility, not modify the old one. I'm adding this to my PR for the new images anyway.

Reply:
I'll probably just close this.